src/atoms/ButtonWithModal.js

```diff
@@ -36,6 +36,8 @@ class ButtonWithModal extends Component {
 		this.state = {
 			modalIsOpen: false,
 		};
+
+		this.toggleModal = this.toggleModal.bind(this);
```

Comment:
This line should not be needed, please remove it

Reply:
weird, without this line I'm getting `state of undefined` error